.github/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ### New features since last release
 
+* The method `Device.supported` that listed all the supported operations and observables
+  was replaced with two separate methods `Device.supports_observable` and `Device.supports_operation`.
+  The methods can now be called with string arguments (`dev.supports_observable('PauliX')`) and with
+  class information arguments (`dev.supports_observable(qml.PauliX)`).
```

Comment:
```suggestion
  class information arguments (`dev.supports_observable(qml.PauliX)`).
  [#276](https://github.com/XanaduAI/pennylane/pull/276)
```

Reply:
I always forget those... I try to remember next time!